src/sfctl/apiclient.py

```diff
@@ -10,20 +10,26 @@
 from azure.servicefabric.service_fabric_client_ap_is import (
     ServiceFabricClientAPIs
 )
-from sfctl.auth import ClientCertAuthentication
-from sfctl.config import (ca_cert_info, cert_info, client_endpoint,
-                          no_verify_setting)
+from sfctl.auth import (ClientCertAuthentication, AdalAuthentication)
+from sfctl.config import (get_config_value, ca_cert_info, cert_info,
```

Comment:
Rather than importing `get_config_value`, write a wrapper function that calls that method that returns the value you're interested in